assets/src/components/editing/elements/list/listActions.tsx

```diff
@@ -86,7 +86,7 @@ export const toggleOrderedList: CommandDescription = {
 const listStyleLabels: Record<OrderedListStyle | UnorderedListStyle, string> = {
   none: 'No Bullet',
   decimal: 'Decimal - 1',
-  'decimal-leading-zero': 'Decimal w/ Zero - 01',
```

Comment:
I also noticed this menu item started wrapping to a second line and looking bad, so reworded it.